packages/react/src/components/Checkbox/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Let's update the other places as well as follows.

```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/156/commits/0f9fa5b6e2dd61aa147c7aa60e68bfd1c97ec6be